test/etaoin/api_test2.clj

```diff
@@ -0,0 +1,24 @@
+(ns etaoin.api-test2
```

Comment:
not the best name I guess but I don't know if it's possible to somehow exclude the test from fixtures: https://github.com/igrishaev/etaoin/blob/3dc31aa2fc4a09056945eb29e35d42fb12799604/test/etaoin/api_test.clj#L33

Reply:
Maybe tests from `api_test.clj` should be moved to something like `functional_test.clj` and thests from `api_test2.clj` should be moved to `api_test.clj`?